docs/manual/manual_advanced_arming.rst

```diff
@@ -0,0 +1,756 @@
+
+User Commands
+**************
+It is possible for users to manually trigger arming and disarming transitions over DroneCAN and IQUART. This approach is generally more difficult to integrate with 
+flight controllers as it requires modifying them to send the arm and disarm commands at the appropriate times, but it is possible for advanced users.
+
+Before attempting to arm manually, be sure to refer to :ref:`manual_arming_throttle_source` for information on how to set the *Manual Arming Throttle Source* appropriately.
+
+DroneCAN
+#########
+The “motor_armed” configuration parameter can be used to query and control the module’s armed state over DroneCAN. Setting this parameter to 0 will trigger a 
+disarming transition, and setting it to 1 will trigger an arming transition. Note that disarming has no effect if you are already disarmed, and arming has 
+no effect if you are already armed. Getting from this parameter will report the current armed state, 0 for disarmed and 1 for armed.
+
+The appearance of this parameter in the DroneCAN GUI Tool is shown below.
+
+.. figure:: ../_static/manual_images/arming/motor_armed_dronecan_gui.png
+    :align: center
+    :width: 60%
+    :alt: motor_armed Parameter
+
+    motor_armed Parameter in DroneCAN GUI Tool
+
+IQUART
+#######
+The “motor_armed” entry of the “arming_handler” client can be used to manually trigger arming and disarming transitions over IQUART. Setting this entry to 
+0 will trigger a disarming transition, and setting it to 1 will trigger an arming transition. Note that disarming has no effect if you are already disarmed, 
+and arming has no effect if you are already armed. Getting from this entry will report the current armed state, 0 for disarmed and 1 for armed.
+
+This functionality is also exposed in the Advanced tab of the IQ Control Center through the *Arming State* parameter as shown below. This entry can be used to check 
+on the current armed state or manually transition the state as needed.
+
+.. figure:: ../_static/manual_images/arming/arming_state_parameter.png
+    :align: center
+    :width: 60%
+    :alt: Arming State Parameter
+
+    Arming State Parameter in IQ Control Center
+
+DSHOT Disarm
+##############
+The :ref:`DSHOT <hobby_dshot>` hobby protocol reserves one of its special values for commanding a disarm. Vertiq modules support this command, and will disarm if a DSHOT disarm 
+command is received.
+
+.. _manual_arming_throttle_source:
+
+Manual Arming Throttle Source
+##############################
+When the module is armed, it must have a specific throttle source set as its armed throttle source, so it knows to reject throttles from other sources for 
+spinning and disarming. See the :ref:`armed_throttle_source_lockout` section for more information on this. 
+
+When the module arms using throttle regions as described in :ref:`throttle_regions`, then the armed throttle source is obvious. When a user manually arms the module 

```

Comment:
while it is obvious, it might be worth spelling out the fact that the arming region way locks on to the throttle type who won the battle for consecutive throttles.